buildstockbatch/gcp/gcp.py

```diff
@@ -610,7 +610,28 @@ def start_batch_job(self, batch_info):
             ),
         )
         instances = batch_v1.AllocationPolicy.InstancePolicyOrTemplate(policy=policy)
-        allocation_policy = batch_v1.AllocationPolicy(instances=[instances])
+        # Make sure the default subnet for this region has access to Google APIs when external IP addresses are blocked.
+        subnet_client = compute_v1.SubnetworksClient()
```

Comment:
It's a bit weird/confusing that this operation is sandwiched within the Batch job definition (I was wondering why this wasn't being added the the AllocationPolicy).

Can this be moved to the top, or maybe even into a validation step?